packages/patternfly-4/react-table/src/components/Table/Table.d.ts

```diff
@@ -60,10 +60,10 @@ export interface ISeparator {
 
 export interface ICell {
   title: String;
-  transfroms: Array<Function>;
-  cellTransforms: Array<Function>;
-  formatters: Array<Function>;
-  cellFormatters: Array<Function>;
+  transfroms?: Array<Function>;
```

Comment:
shouldn't it be `transforms` ?

Reply:
the existing property was named transfroms - we should probably fix the naming as part of this breaking change right now

Reply:
Fixed it!